internal/cloud/e2e/main_test.go

```diff
@@ -66,6 +68,96 @@ func setup() func() {
 		teardown()
 	}
 }
+func testRunner(t *testing.T, cases testCases, orgCount int, tfEnvFlags ...string) {
+	for name, tc := range cases {
+		tc := tc // rebind tc into this lexical scope
+		t.Run(name, func(subtest *testing.T) {
+			subtest.Parallel()
+
+			orgNames := []string{}
+			for i := 0; i < orgCount; i++ {
+				organization, cleanup := createOrganization(t)
+				defer cleanup()
+				orgNames = append(orgNames, organization.Name)
+			}
+
+			exp, err := expect.NewConsole(defaultOpts()...)
+			if err != nil {
+				subtest.Fatal(err)
+			}
+			defer exp.Close()
+
+			tmpDir, err := ioutil.TempDir("", "terraform-test")
+			if err != nil {
+				subtest.Fatal(err)
+			}
+			defer os.RemoveAll(tmpDir)
+
+			tf := e2e.NewBinary(terraformBin, tmpDir)
+			tfEnvFlags = append(tfEnvFlags, "TF_LOG=INFO")
+			tfEnvFlags = append(tfEnvFlags, cliConfigFileEnv)
+			for _, env := range tfEnvFlags {
+				tf.AddEnv(env)
+			}
+			defer tf.Close()
+
+			var orgName string
+			for index, op := range tc.operations {
+				if len(orgNames) == 1 {
+					orgName = orgNames[0]
+				} else {
+					orgName = orgNames[index]
```

Comment:
Am I right in thinking that len(cases) and orgCount always have to be the same? I find this interface a little confusing, but maybe you can simplify it because we always call `testRunner` with orgCount = 1 ?

Reply:
There is at least 1 test that uses multiple organizations, but you are correct that `orgCount` is already known so we don't need to recalculate it.

Reply:
ah yeah that diff was hidden by github when I looked. I don't think we need to remove orgCount, because then tests with multiple operations would never share the same org... maybe there is another function or parameter like `testRunnerDistinctOrgs` flavored function or `distinctOrgs bool` parameter

Reply:
I changed the code to use `orgCount` on line 106 instead of recalculating the count. Do you think an additional boolean function would be valuable?